pkgs/build-support/emacs/generic.nix

```
@@ -56,6 +56,18 @@ stdenv.mkDerivation ({
   meta = defaultMeta // meta;
 }
 
+// lib.optionalAttrs emacs.nativeComp {
```

```suggestion
// lib.optionalAttrs (emacs.nativeComp or false) {
```

for emacs from older nixpkgs

---

Good catch, thanks.